### PR TITLE
Add mUSDK and mUSDT

### PR DESCRIPTION
**Pull request by coreycaplan3**

### CODE REVIEW

**Comment by catenocrypt** [comment]
Please add an `info.json` files, one next to each `logo.png` file, with info about the project, of the form:

```json
{
    "name": "Trust Wallet Token",
    "website": "https://trustwallet.com",
    "short_description": "Utility token to increase adoption of cryptocurrency.",
    "explorer": "https://etherscan.io/token/0xA866F0198208Eb07c83081d5136BE7f775c2399e",
    "socials": [
        {
            "name": "Twitter",
            "url": "https://twitter.com/trustwalletapp",
            "handle": "TrustWalletApp"
        },
        {
            "name": "Reddit",
            "url": "https://reddit.com/r/trustapp",
            "handle": "TrustApp"
        }
    ]
}
```

**Comment by coreycaplan3** [comment]
Just updated the PR with the requested changes!

**Comment by catenocrypt** [comment]
1., Can you please fix the website -- double com at the end   "website": "https://defimoneymarket.com.com",
2.,  These wrapped tokens have very low circulations.  Could you share some link about DMM numbers?  (liquidity, volume, ...)

File: blockchains/ethereum/allowlist.json [hunk 1]
```diff
@@ -1603,6 +1603,7 @@
     "0x4119876899a9B7D92ECF9DFB9479962499DD5B48",
     "0x411A81C8Cd604941be13dBBD33B54e6f25d84683",
     "0x4129D3b7a6A2c5C997774077aC02bDafd1AF1d6a",
+    "0x413237cf1b1Ee88e551b5ad1DC117aBF1004F9eA",
```
Comment (vikmeup) [inline blockchains/ethereum/allowlist.json:1606]: can you remove from this link? it's generated automatically

**Review by vikmeup — COMMENTED** [review]
Can you fix CI and should be good to merge

File: blockchains/ethereum/assets/0xD1670Cb7c01987a8DdD6976C4894463460Eeb8a2/info.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "DMM: USDK",
+    "website": "https://defimoneymarket.com.com",
```
Comment (Iamdeadlyz) [inline blockchains/ethereum/assets/0xD1670Cb7c01987a8DdD6976C4894463460Eeb8a2/info.json:3]: Please change `https://defimoneymarket.com.com` to `https://defimoneymarket.com`

File: blockchains/ethereum/assets/0x84d4AfE150dA7Ea1165B9e45Ff8Ee4798d7C38DA/info.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "DMM: USDT",
+    "website": "https://defimoneymarket.com.com",
```
Comment (Iamdeadlyz) [inline blockchains/ethereum/assets/0x84d4AfE150dA7Ea1165B9e45Ff8Ee4798d7C38DA/info.json:3]: Please change `https://defimoneymarket.com.com` to `https://defimoneymarket.com`

**Review by Iamdeadlyz — CHANGES_REQUESTED** [review]

**Comment by coreycaplan3** [comment]
The stats for our overall usage can be found here:
https://app.defimoneymarket.com/
https://defimarketcap.io/protocol/dmm

We have pull requests open for them to track more wrapped tokens. The two we added have a 6-figure market cap.

**Review by Iamdeadlyz — APPROVED** [review]

**Pull request merged**